src/main/java/frc/robot/commands/RampMove.java

```diff
@@ -0,0 +1,36 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.Ramp;
+
+public class RampMove extends Command{
+    
+    private Ramp ramp;
+    private double degrees;
+
+    public RampMove(Ramp ramp, double degrees) {
+        this.ramp = ramp;
+        this.degrees = degrees;
+        addRequirements(this.ramp);
+    }
+
+    @Override
+    public void initialize() {
+        super.initialize();
```

Comment:
You don't need to call super.initialize() here, the base class for Command doesn't do anything.